app/Rom.php

```diff
@@ -584,6 +584,9 @@ public function setGanonInvincible(string $setting = 'no'): self
             case 'custom':
                 $byte = pack('C', 0x04);
                 break;
+            case 'ganonhunt':
+                $byte = pack('C', 0x05);
+                break;
```

Comment:
`custom` will need to be updated to `crystals_only` where ever this is called.

```suggestion
            case 'crystals_only':
                $byte = pack('C', 0x04);
                break;
            case 'ganonhunt':
                $byte = pack('C', 0x05);
                break;
            case 'lightspeed':
                // light world only, pull ped, kill aga 1
                $byte = pack('C', 0x06);
                break;
            case 'crystals_bosses':
                $byte = pack('C', 0x07);
                break;
            case 'bosses_only':
                $byte = pack('C', 0x08);
                break;
            case 'dungeons_no_agahnim':
                // all dungeons, aga 1 not required
                $byte = pack('C', 0x09);
                break;
```